packages/storybook/src/stories/Filters/molecules/DatePicker.stories.tsx

```diff
@@ -24,29 +24,58 @@ const exampleCallback = <T extends Function>(fn: T): T => {
   return fn;
 };
 
+const START_RANGE: Date = new Date();
+START_RANGE.setDate(0);
+START_RANGE.setDate(START_RANGE.getDate() - 20);
+const END_RANGE: Date = new Date();
+END_RANGE.setDate(1);
+END_RANGE.setDate(END_RANGE.getDate() + 70);
+
+
+const TODAY: Date = new Date();
+const START: Date = new Date();
+const END: Date = new Date();
+START.setDate(TODAY.getDate() - 4);
+END.setDate(TODAY.getDate() + 4);
+
+
+const initialValue = {
+  start: START,
+  end: END
+}
+const datesRange = {
+  start: START_RANGE,
+  end: END_RANGE
+}
+
 export const _DatePicker = () => {
   const language = select('Language', { English: 'en', Japanese: 'ja' }, 'ja');
+  const initialValueObj = object('Initial Value', initialValue);
   const dateMode = select('Date Mode', { single: 'single', interval: 'interval' }, 'interval');
   const timeMode = select('Time Mode', { off: 'off', single: 'single', interval: 'interval' }, 'interval');
   const dateTimeTextUpper = text('Date Time Text Upper', 'From');
   const dateTimeTextLower = text('Date Time Text Lower', 'To');
   const timeZoneTitle = text('Time Zone Title', 'Timezone');
   const timeZoneValueTitle = text('Time Zone Value', 'JST');
   const updateCallback = action('Date / Time Updated');
+  const sendRange = boolean('Send Available Range', false);
+  const availableRangeDates = object('Available Range', datesRange);
```

Comment:
Can we enable this by default and make the default date range something that would be more visible without needing pagination? 

I think as one of our most common use cases is recently data, having a period that is 2 weeks before until today's date would be good if we're keeping the date dynamic.